src/util/fields.py

```diff
@@ -855,6 +856,32 @@ def _update_field_usage_multi(cr, models, old, new, domain_adapter=None, skip_in
     }
 
     # ir.action.server
+    if column_exists(cr, "ir_act_server", "update_path") and only_models:
+        cr.execute(
+            """
+            SELECT a.id,
+                   a.update_path,
+                   m.model
+              FROM ir_act_server a
+              JOIN ir_model m
+                ON a.model_id = m.id
+             WHERE a.state = 'object_write'
+               AND a.update_path ~ %s
+            """,
+            [r"\y{}\y".format(re.escape(old))],
```

Comment:
Consistence with other queries in this function
```suggestion
               AND a.update_path ~ %(old)s
            """,
            p,
```

Or at least (but less readable)

```suggestion
               AND a.update_path ~ %s
            """,
            [p["old"]],
```